content/using/running/minio.md

```diff
@@ -0,0 +1,176 @@
+  -e "MINIO_ACCESS_KEY=<access_key>" \
+  -e "MINIO_SECRET_KEY=<secret_key>" \
+  -e "MINIO_DOMAIN=s3.example.com" \
+  minio/minio server /data
+```
+
+`/mnt/data` is the path where your uploaded data will be stored on your host machine - you can change this as necessary.
+
+Be sure to add the `MINIO_DOMAIN` environment variable; this tells MinIO to accept virtual host style URLs (`BUCKET.s3.example.com` rather than `s3.example.com/BUCKET`), which are required for compatibility with Urbit.
+
+Your access key and secret key can be anything of your choosing — make sure they're secure! ~ribben-donnyl recommended a handy tool called [minio-keygen](https://github.com/iwittkau/minio-keygen) that will generate secure keys for you. You could also use your `@p` and `+code` for an easy to remember solution.
+
+### Create DNS records
+
+Now, you'll need to point your own domain at your MinIO installation. Via your domain's DNS settings (usually configured on the registrar you bought your domain through), create two `A` records:
+
+- `s3`.example.com, and
+- `*.s3`.example.com
+
+Both should point at the IP address of your host machine. If you are hosting on your own hardware, this could require port-forwarding via your router so that your host machine is reachable from outside of your home network.
+
+It's important that you create both records — without the wildcard, you'll be able to access your S3 web interface but not upload/download content from any buckets you create.
+
+DNS records can take a little while to propagate, so don't worry if you type your new URL into your browser and don't see anything yet.
+
+### Configure a TLS certificate
+
+Next, you need to aquire a TLS certificate from Let's Encrypt. TLS ensures that all traffic to and from your S3 installation is encrypted, and is required for your installation to work with Urbit.
+
+The EFF Certbot tool is the easiest way to get a certificate, and [this page](https://certbot.eff.org/instructions) will tell you how to install it depending on your requirements. Select **NGINX** and your operating system for the correct guide.
+
+Once Certbot is installed, you can generate a certificate with the following command:
+
+```
+sudo certbot --server https://acme-v02.api.letsencrypt.org/directory \
+  -d *.s3.example.com \
+  -d s3.example.com \
+  --manual --preferred-challenges dns-01 certonly
+```
+
+After answering a few questions, you will be prompted with a DNS challenge to prove that you own the domains you are creating certificates for. After successfully completing the DNS challenge, your certificate files will be generated and Certbot will print the paths at which they are located. Take note of these for the next step.
+
+Let's Encrypt certificates expire after 90 days, so don't forget to renew them. You should receive an email (to the address you entered when creating the certs) before they are due to expire. To renew, simply repeat the process listed above.
+
+### Install NGINX
```

Comment:
It would be nice to also have a guide on how to do this with `caddy`, since its simpler than `nginx`.